source/user-manual/ruleset/testing.rst

```diff
@@ -1,61 +1,484 @@
+| Endpoint                      | Method          | Description                                                           |
++===============================+=================+=======================================================================+
+| ``/logtest``                  | PUT             | Check if an alert matches a log and query the related information.    |
++-------------------------------+-----------------+-----------------------------------------------------------------------+
+| ``/logtest/sessions/{token}`` | DELETE          | Delete the session corresponding to ``{token}``                       |
++-------------------------------+-----------------+-----------------------------------------------------------------------+
+
+``PUT /logtest`` accepts the following list of parameters as a *RequestBody*:
+
+-  **token**: alphanumeric string
+-  **log_format**: ``syslog`` or ``json``
+-  **location**: path string
+-  **event**: string
+
+Logging into the Wazuh API
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Wazuh API endpoints require authentication. All calls must include a JSON Web Token. You can use the cURL command to log in. The Wazuh API provides a JWT token upon success.
+
+#. Run the following command replacing ``<WAZUH_API_USER>`` and ``<PASSWORD>`` with your own values:
+
+   .. code-block:: bash
+
+      TOKEN=$(curl -u <WAZUH_API_USER>:<PASSWORD> -k -X POST "https://localhost:55000/security/user/authenticate?raw=true")
+
+#. Check that everything works correctly.
+
+   .. code-block:: bash
+
+      curl -k -X GET "https://localhost:55000/?pretty=true" -H "Authorization: Bearer $TOKEN"
+
+   .. code-block:: JSON
+      :class: output
+
+      {
+         "data": {
+            "title": "Wazuh API REST",
+            "api_version": "4.4.1",
+            "revision": 40406,
+            "license_name": "GPL 2.0",
+            "license_url": "https://github.com/wazuh/wazuh/blob/v4.4.1/LICENSE",
+            "hostname": "centos7",
+            "timestamp": "2023-04-25T13:39:23Z"
+         },
+         "error": 0
+      }
+
+First request
+^^^^^^^^^^^^^
+
+The first time you send a processing request, it has no logtest session token. Since there is no active session, a processing
+log request is sent to logtest in Analysisd.
+
+Use the following sample data for request:
+
++------------------+--------------------------------------+----------------------------------------------------------------------------------------------+
+| Field            | Description                          | Example                                                                                      |
++==================+======================================+==============================================================================================+
+| log_format       | Type of log, ``syslog`` or ``json``  | ``syslog``                                                                                   |
```

Comment:
The [Logtest documentation](https://documentation.wazuh.com/current/user-manual/api/reference.html#operation/api.controllers.logtest_controller.run_logtest_tool) indicates that the endpoint accepts the following log formats: syslog, json, snort-full, squid, eventlog, eventchannel, audit, mysql_log, postgresql_log, nmapg, iis, command, full_command, djb-multilog, multi-line.